src/game/mod.rs

```diff
@@ -7,17 +7,47 @@ pub mod world;
 use graphics::renderer::Renderer;
 use util::vector::Vector2D;
 
-use self::console::{Console, CONSOLE_WIDTH};
+use self::console::Console;
 use self::game_state::{state_explore::StateExplore, GameState};
 
 use std::io::{stdin, stdout, Write};
 
-pub const GAME_AREA_SIZE: Vector2D<i32> = Vector2D { x: 81, y: 45 };
+pub const SCREEN_SIZE: Vector2D<i32> = Vector2D { x: 120, y: 52 };
```

Comment:
Only issue I have now is that screen size is no longer widened when the console is widened :(

Reply:
Ideally, game is the left side, console is right side - so width of the game would be the width of the game area + width of the console

Reply:
If you want, I can make game area size constant and the screen size dynamic

Reply:
That would be better, thanks 😄

Reply:
Hmmm, maybe it would be better to keep existing code if the game will resize when the terminal window resizes

Reply:
I don't there is anything we can do about terminal window resizing :(

Reply:
https://github.com/eminence/terminal-size

Reply:
If you don't want this:
> I can make game area size constant and the screen size dynamic

Reply:
Crate seems to just detect size, but doesn't detect resizing (I assume this is all platform dependent anyway).

> I can make game area size constant and the screen size dynamic

By dynamic, do you mean just work it from the game area constant/ console size constant?

Reply:
Yep

Reply:
Ahh yeah perfect then, thanks

Reply:
Also, user can specify screen dimensions via CLI option

Reply:
Is this referring to the crate?

Reply:
Oops, didn't mean to start a review :S

Reply:
No, I'm saying that about the game